package/resource_monitor/resource_monitor/overlay/etc/logrotate.d/extrace_log

```diff
@@ -0,0 +1,7 @@
+/var/log/extrace.log {
```

Comment:
This has moved to `/var/run/resource_monitor`

Reply:
removed